package/yast2-ntp-client.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jun 30 06:51:52 UTC 2016 - jreidinger@suse.com
+
+- Fix regression: crash when reading cron entry failed
+  ( discovered by sap-ha module ) (bsc#977610)
```

Comment:
The bug only describes the original problem, not the one that we are fixing now.
Please add a description of the new problem, otherwise we risk making a wrong fix.